src/main/java/duke/pack/Deadline.java

```diff
@@ -0,0 +1,48 @@
+package duke.pack;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected String time;
+    protected LocalDate date;
+    protected String fullDesc;
+
+    /**
+     * creates a deadline type of task
+     * @param description the task to be done
+     * @param time the time the task is due
+     * @param date the date the task is due
+     * @param fullDesc the full description of task
+     */
+    public Deadline(String description, String time, LocalDate date, String fullDesc) {
+        super(description, fullDesc);
```

Comment:
If all subclasses of Task contains a fullDesc, then you can move the declaration to the parent class instead of repeating in every child class